samples/nrf9160/at_cmd_parser/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright (c) 2018 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+cmake_minimum_required(VERSION 3.8.2)
+
+include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
+project(NONE)
```

Comment:
Let's give it a name.

Reply:
AT_CMD_DECODER_DEMO?

Reply:
Better give it the same name as the folder (and sample) : `at_cmd_parser`.

Reply:
ok